educational_modules/forms/app/covid_education.properties.json

```diff
@@ -2,7 +2,7 @@
     "icon": "icon-healthcare",
```

Comment:
This PR contains a new file `.DS_Store` which shouldn't be committed to the repository.
Can you remove it from the branch and add `.DS_Store` into the `.gitignore` file?

Reply:
Removed from fs and added to .gitignore

Reply:
Nice job! Looks like you deleted one, but there is one more remaining at  `educational_modules/forms/app/.DS_Store`